projects/angular/src/layout/_login.clarity.scss

```
@@ -183,7 +185,7 @@
             -webkit-mask-size: cover;
             mask-size: cover;
             background-color: login-variables.$clr-login-error-icon-color;
-            margin: tokens.$cds-global-space-1 tokens.$cds-global-space-4 0 0;
+            margin: tokens.$cds-global-space-1 density.$clr-base-horizontal-offset-s 0 0;
```

Try use density `layout-4xs`
```suggestion
            margin: density.$clr-base-layout-space-4xs density.$clr-base-horizontal-offset-s 0 0;
```